pkg/client/listers/core/internalversion/expansion_generated.go

```diff
@@ -58,6 +58,10 @@ type LimitRangeNamespaceListerExpansion interface{}
 // NamespaceLister.
 type NamespaceListerExpansion interface{}
 
+// NodeListerExpansion allows custom methods to be added to
+// NodeLister.
+type NodeListerExpansion interface{}
```

Comment:
This is autogenerated code, generated by lister-gen.

Reply:
Yep. Generated it by the script.